### ISSUE TITLE
'=' used outside of function results in incorrect start line 

### ISSUE THREAD

**Issue by jwons** [End-to-end-provenance/RDataTracker#411]
In the following script:
```{r}
y <- 5
x <- 10
 
foo = 7
bar = 9
```
When ddg.run is called the start line for 'foo' and 'bar' is both set to 4. The start column for 'bar' will also be something other than 1 despite being the first expression on a line. When the script is read in using readLines() these lines are separated. Changing the '=' to '<-' stops this behavior. 

prov.json procedure nodes of 'foo' and 'bar' from this script:
```{json}
"rdt:p4": {
	"rdt:name": "foo = 7",
	"rdt:type": "Operation",
	"rdt:elapsedTime": 12.501,
	"rdt:scriptNum": 0,
	"rdt:startLine": 4,
	"rdt:startCol": 1,
	"rdt:endLine": 4,
	"rdt:endCol": 3
},
"rdt:p5": {
	"rdt:name": "bar = 9",
	"rdt:type": "Operation",
	"rdt:elapsedTime": 12.507,
	"rdt:scriptNum": 0,
	"rdt:startLine": 4,
	"rdt:startCol": 7,
	"rdt:endLine": 4,
	"rdt:endCol": 7
}
```

**Comment by blernermhc** [comment]
This looks like a bug in parseData from the utils package.  I submitted a bug to r-devel@r-project.org on July 30, 2018, saying this:

I have run into a problem with parseData from the utils package.  When an assignment is done with = instead of <-, the information provided by parseData does not include an entry for the assignment.

For this input, stored in file "BadPosition.R":
```
y <- 5
foo = 7
```
And running this code:
```
parsed <- parse("BadPosition.R", keep.source=TRUE)
parsedData <- utils::getParseData (parsed, includeText=TRUE)
print(paste("parseData =", parsedData))
```
I get the following output:
```
[1] "parseData = c(1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2)"                                                                                                      
[2] "parseData = c(1, 1, 1, 3, 6, 6, 1, 1, 5, 7, 7)"                                                                                                      
[3] "parseData = c(1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2)"                                                                                                      
[4] "parseData = c(6, 1, 1, 4, 6, 6, 3, 3, 5, 7, 7)"                                                                                                      
[5] "parseData = c(7, 1, 3, 2, 4, 5, 10, 12, 11, 13, 14)"                                                                                                 
[6] "parseData = c(0, 3, 7, 7, 5, 7, 12, 0, 0, 14, 0)"                                                                                                    
[7] "parseData = c(\"expr\", \"SYMBOL\", \"expr\", \"LEFT_ASSIGN\", \"NUM_CONST\", \"expr\", \"SYMBOL\", \"expr\", \"EQ_ASSIGN\", \"NUM_CONST\", \"expr\")"
[8] "parseData = c(FALSE, TRUE, FALSE, TRUE, TRUE, FALSE, TRUE, FALSE, TRUE, TRUE, FALSE)"                                                                
[9] "parseData = c(\"y <- 5\", \"y\", \"y\", \"<-\", \"5\", \"5\", \"foo\", \"foo\", \"=\", \"7\", \"7\")"                                                
```
Notice how there is an entry for "y <- 5" beginning on line 1, column 1, ending at line 1, column 6, but there is no analogous entry for "foo = 7".

I am running R 3.5.0 on a Mac running 10.12.6.

**Comment by blernermhc** [comment]
R developers responded that this is a bug in their code.

**Comment by blernermhc** [comment]
See issue 472 for examples of this occurring in our script tests.

**Comment by blernermhc** [comment]
Fixed in R 3.6.0
